### PR TITLE
Integration

### PR DESCRIPTION
**Pull request by vytautassugintas**

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -20,9 +19,9 @@ module.exports = function (config) {
             'app/*.module.js',
             'app/*.controller.js',
             'app/*.config.js',
-            'app/*.utils.js',
             'app/kudos-feed/kudos-feed.controller.js',
             'app/components/**/*.component.js',
+            'app/components/**/*.controller.js',
```
Comment (Medds) [inline karma.conf.js]: If you chose to have tests in a standalone folder - might as well make the runner just take ALL the *.js files available.

File: src/app/components/kudos-top-receivers/kudos-top-receivers.component.js [hunk 2]
```diff
@@ -1,46 +1,38 @@
...
 
-        $scope.userReceivedKudos = 0;
-        $scope.topReceivers = [];
-        $scope.acornPlural = acornPlural;
-        $scope.setTopReceivers = setTopReceivers;
+    angular
+        .module('myApp.components.topReceivers', [])
+        .component('kudosTopReceivers', {
+            templateUrl: 'app/components/kudos-top-receivers/kudos-top-receivers.html',
+            controller: ('TopReceiversController', TopReceiversController),
```
Comment (Medds) [inline src/app/components/kudos-top-receivers/kudos-top-receivers.component.js]: Hmm?

File: src/app/components/kudos-top-senders/kudos-top-senders.component.js [hunk 3]
```diff
@@ -1,9 +1,19 @@
...
-    var TopSendersController = function ($scope, ProfileService, $httpParamSerializer, Resources) {
-        $scope.topSenders = [];
-        $scope.acornPlural = acornPlural;
-        $scope.setTopSenders = setTopSenders;
+    TopSendersController.$inject = ['ProfileService', '$httpParamSerializer', 'Resources'];
+
+    angular.module('myApp.components.topSenders', [])
+        .component('kudosTopSenders', {
+            templateUrl: 'app/components/kudos-top-senders/kudos-top-senders.html',
+            controller: ('TopSendersController', TopSendersController),
```
Comment (Medds) [inline src/app/components/kudos-top-senders/kudos-top-senders.component.js]: Hmm?

File: src/app/settings/settings.view.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
+<title>Profile</title>
```
Comment (Medds) [inline src/app/settings/settings.view.html]: Don't use `<title>` outside of `<head>`

**Comment by MindaugasJaunius** [comment]
Hack?

**Pull request merged**